code/modules/mob/living/carbon/human/human_emote.dm

```diff
@@ -31,6 +31,31 @@
 			return TRUE
 	return ..()
 
+/datum/emote/living/carbon/human/hiss
+	key = "hiss"
+	key_third_person = "hisses"
+	message = "hisses."
+	message_mime = "hisses silently."
+	message_param = "hisses at %t."
+	emote_type = EMOTE_AUDIBLE | EMOTE_MOUTH
+	muzzled_noises = list("weak hissing", "irritated")
+
+/datum/emote/living/carbon/human/hiss/get_sound(mob/living/user)
+	var/mob/living/carbon/human/H = user
+	if(H.mind?.miming || !istype(H))
+		return
+	else
+		return H.dna.species.hiss_sound
+
+/datum/emote/living/carbon/human/hiss/select_message_type(mob/user, msg, intentional) //Only used when emote forced by burn wounds
+	. = ..()
+	var/mob/living/carbon/human/H = user
+	if(intentional == FALSE)
+		if(H.mind?.miming)
+			return "grimaces as their skin peels off!"
```

Comment:
Honestly, I think you might want to make a subtype of hiss with a slightly different key that gets called for this. Something like burn_hiss. I think it's entirely reasonable that we might want hiss to get called unintentionally without giving the flavor that their flesh is peeling off.

Reply:
Good point - I originally wanted to make it a subtype but just not sure if that would bloat the file too much.

Reply:
Code quality is much more important than worrying about file bloat, especially in a file that's already as big as this

Reply:
Changes now in newest commit